SUMO-236061: Azure Append Blob Collection (#3979)
SumoLogic/sumologic-documentation: blog-service/2019/12-31.md

---
## February 22, 2019 (Apps)

> Update - The [Azure Network Watcher App](/docs/integrations/microsoft-azure/network-watcher) and [Azure Blob Storage app](/docs/send-data/collect-from-other-data-sources/azure-blob-storage/block-blob/collect-logs) have an enhanced collection processes. The SumoLogic App for Azure Network Watcher leverages Network Security Group (NSG) flow logs to provide real-time visibility and analysis of your Azure Network. The Sumo Logic App for Azure Blob Storage is a Sumo integration that provides a event-based pipeline for shipping monitoring data from Azure Blob Storage to an HTTP source on Sumo Logic.

Reviewer: ```suggestion
Update - The Sumo Logic apps for [Azure Network Watcher](/docs/integrations/microsoft-azure/network-watcher) and [Azure Blob Storage](/docs/send-data/collect-from-other-data-sources/azure-blob-storage/block-blob/collect-logs) now have an enhanced collection processes. The Azure Network Watcher app leverages Network Security Group (NSG) flow logs to provide real-time visibility and analysis of your Azure Network. The Sumo Logic App for Azure Blob Storage provides an event-based pipeline for shipping and monitoring data from Azure Blob Storage to an HTTP source on Sumo Logic.
```
